PowerShellTools.Test/IntelliSense/CommentAreaUnitTests.cs

```diff
@@ -73,7 +73,7 @@ public void TestNormalCommentScript()
             IsInCommentAreaTestHelper(script, 10, true);
             IsInCommentAreaTestHelper(script, 100, false);
             IsInCommentAreaTestHelper(script, 300, true);
-            IsInCommentAreaTestHelper(script, script.Length, true);
+            IsInCommentAreaTestHelper(script, script.Length - 1, true);
         }
 

```

Comment:
Do we have a test to validate and prevent regression of the specific instance of #497 ?